governance/polls/Ratification Poll - Modify Oracle Core Unit Budget (ORA-001) - August 8, 2022.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Ratification Poll for Modify Oracle Core Unit Budget (MIP40c3-SP75) - August 8, 2022 
+summary: MIP40c3-SP75 renews the Oracle Core Unit ORA-001 budget from July 1, 2022 through June 30, 2023.
+discussion_link: https://forum.makerdao.com/t/mip40c3-sp75-modify-oracles-core-unit-budget-ora-001/16397
+parameters:
+    input_format: rank-free
+    victory_conditions:
+        - { type : instant-runoff }
+    result_display: instant-runoff-breakdown
+version: v2.0.0
+options:
+   0: Abstain
+   1: Approve existing budget (2021-2022) (Business as usual)
+   2: Approve proposed budget (2022-2023) (Reduced budget exempt from further cuts for 12 months)
+   3: Reject Budget
+start_date:2022-08-08T16:00:00
```

Comment:
add space - I think this is breaking the formatting - can be seen on rich diff view
```suggestion
start_date: 2022-08-08T16:00:00
```